src/fpylll/gso.pyx

```diff
@@ -13,6 +13,11 @@ from util cimport preprocess_indices, check_float_type
 from fpylll cimport mpz_double, mpz_ld, mpz_dpe, mpz_mpfr, mpz_dd, mpz_qd
 from fplll cimport FT_DOUBLE, FT_LONG_DOUBLE, FT_DD, FT_QD, FT_DPE, FT_MPFR, FloatType
 
+import cython
+import numpy as np
```

Comment:
My current idea is to return a `vector<double>` in fplll and then have some code like:

```
def foo(dest, int n):
    cdef int i, j
    cdef vector[double] src    
    for i in range(n): # C loop
        for j in range(n): # C loop
            dest[i,j] = src[i+j*n] # C read, Python write
```

in fpylll.